File: week-02/day-01/diagonal.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %%  %
# % % %
# %  %%
# %   %
# %%%%%
#
# The square should have as many lines as the number was

number = int(input('enter number: '))
space = " "
percent = '%'
a = 0
print(percent * number + 2 * percent)

while a < number:
  print(percent + space * a + percent + (((number - a - 1) * space) + percent))

  a += 1

print(percent * number + 2 * percent)

File: week-07/day-01/diagonal.js
'use strict';

var lineCount = 6;


// Write a program that draws a
// square like this:
//
//
// %%%%%
// %%  %
// % % %
// %  %%
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is

var draw = '%';
var space = ' ';
var length = 10;

console.log(draw.repeat(length + 2));
for(var i = 0; i < length; i++){
  console.log(draw + space.repeat(i) + draw + space.repeat(length - i - 1) + draw);
}
console.log(draw.repeat(length + 2));